posthog/api/decide.py

```diff
@@ -160,11 +162,19 @@ def get_decide(request: HttpRequest):
 
             if api_version == 2:
                 response["featureFlags"] = active_flags
-            elif api_version == 3:
+            elif api_version >= 3:
                 # v3 returns all flags, not just active ones, as well as if there was an error computing all flags
                 response["featureFlags"] = feature_flags
                 response["errorsWhileComputingFlags"] = errors
                 response["featureFlagPayloads"] = feature_flag_payloads
+                if api_version == 4:
+                    # v4 returns the feature previews as well
```

Comment:
I stuck to `/decide` mainly because this involved the least amount of library changes + adding a new endpoint may or may not involve some infra-level routing concerns (would this go to the `web` or `events` pods 🤔) + also I strongly dislike how all these endpoints (decide/capture/e/engage/s) are at the top level, potentially conflicting with app routes.

I agree this should be separate though, this data should be loaded on request.

Reply:
Makes sense, agree